e2e2/test/images/bert-inference/Dockerfile

```diff
@@ -0,0 +1,31 @@
+# Use the NVIDIA CUDA runtime as a parent image
+FROM nvidia/cuda:12.5.0-devel-ubuntu22.04
+
+# Set environment variable to disable interactive prompts
+ENV DEBIAN_FRONTEND=noninteractive
+
+# Install Python 3.11
+RUN apt-get update && apt-get install -y \
+    software-properties-common && \
+    add-apt-repository ppa:deadsnakes/ppa && \
```

Comment:
@mattcjo @weicongw what are your opinions of different ways of installing dependencies like python in these images? I see weicong's is building from source in the neuron image  https://github.com/aws/aws-k8s-tester/blob/cc66356a28d11fdd4a60573d2a2bbe502a14dbab/e2e2/test/images/neuron/Dockerfile#L90-L101

do you see a benefit in being consistent for all our images?

Reply:
So, in theory I prefer installing directly from source as that gives us the most control and consistency. My main issue with it though is that you need to specify the minor version as well. I feel like using the system package managers do just fine, but would like to hear everyone's take

Reply:
One other thing to add on... My main point in support of installing from source is that there are sometime new features and optimizations introduced in minor releases. This has the potential of leading to different performance results between minor versions. The likelihood of this happening, or being of any significance? I think pretty low, but something worth noting

Reply:
+1 to those points, and irrespective of execution i think having shared expectations across our images is also a good idea (unless there are obvious requirements software/hardware differences)

Reply:
Sounds good to me. I'll make the changes now

Reply:
Done